src/components/currencies/currencyListing.js

```diff
@@ -0,0 +1,66 @@
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { v4 as uuidv4 } from 'uuid';
+import Grid from '@mui/material/Grid';
+
+import getRestructuredObject from '../../common/dataLogic';
+import ActionAreaCard from '../currenciesCard/currenciesCard';
+import Paginate from '../paginate/paginate';
+import './currencyListing.scss';
+
+const CurrencyListing = () => {
```

Comment:
- [ ] Nice use of the pagination feature 💯. However, I strongly suggest that you also add a search feature which will ensure that the currency list can be filtered using given parameters, e.g. currency name. This will improve the user experience. Imagine having the pagination and search features, your app will be awesome. Please give it a try.

Reply:
I  intended to implement a search input, an input that updated the base currency and input to convert currencies but I kept on having bugs that made me repeat the week. I will make sure I implement them on my own time.